### PR TITLE
feat: clean up node operator overview

### PR DESCRIPTION
**Pull request by smartcontracts**
Small clean up to the Node Operator overview page.

### CODE REVIEW

File: pages/builders/node-operators/overview.mdx [hunk 1]
```diff
@@ -4,54 +4,61 @@ lang: en-US
...
+Legacy Geth is **not** required and is typically only necessary if you want to maintain a complete archive node for OP Mainnet.
 
-You can also compile `op-geth` and `op-node` from source.
+## Software Releases
 
-## Node Operator Tutorials
+It's important to keep your node software up to date to ensure that you won't fall out of sync with other OP Mainnet nodes when there's an upgrade.
+Software updates can also include important bug fixes and patches that can help keep your node stable.
+Refer to the [Software Releases](/builders/node-operators/releases) page for a detailed look at the latest releases of various Rollup Node and Execution Client implementations.
+Notifications about important updates are also posted to the [Optimism Twitter account](https://twitter.com/optimismPBC).
```
Comment (sbvegan) [inline pages/builders/node-operators/overview.mdx:45]: This twitter account is deprecated. On foundation websites, we link out to https://twitter.com/optimism

File: pages/builders/node-operators/overview.mdx [hunk 2]
```diff
@@ -4,54 +4,61 @@ lang: en-US
...
+The Execution Client is responsible for executing the block payloads it receives from the Rollup Node over JSON-RPC via the standard [Ethereum Engine API](https://github.com/ethereum/execution-apis/blob/main/src/engine/common.md#engine-api----common-definitions).
+The Execution Client exposes the standard JSON-RPC API that Ethereum developers are familiar with, and can be used to query blockchain data and submit transactions to the network.
+The Execution Client is largely analogous to an [execution client](https://ethereum.org/en/developers/docs/nodes-and-clients/#what-are-nodes-and-clients) in Ethereum.
 
-## Getting the Software
+### Legacy Geth
 
-`op-geth` and `op-node` are available as Docker images. Consult the [documentation](/chain/networks) for the network you are participating in to get the correct image tag.
+OP Mainnet underwent a large database migration as part of the [Bedrock Upgrade](https://blog.oplabs.co/introducing-optimism-bedrock/) in 2023.
```
Comment (sbvegan) [inline pages/builders/node-operators/overview.mdx:32]: might be worth it to link this as well: https://blog.oplabs.co/reproduce-bedrock-migration/

File: pages/builders/node-operators/overview.mdx [hunk 3]
```diff
@@ -4,54 +4,61 @@ lang: en-US
...
 
-For OP Mainnet (but not OP Sepolia) you may also need to run a third component called Legacy Geth. Legacy Geth is used to serve execution traces for transactions prior to the Bedrock upgrade, which OP Mainnet refers to as "historical transactions". When the Rollup Node encounters an RPC call that requires historical execution traces, it will forward the request to Legacy Geth. Note, however, that unlike our previous networks requests for historical *data* will be served by the Execution Engine directly. This distinction will be clarified later on in this document.
+## Node Architecture
 
-The architecture of a typical Bedrock deployment looks like this:
+All OP Mainnet nodes are composed of two core software services, the Rollup Node and the Execution Client.
+OP Mainnet also optionally supports a third component, Legacy Geth, that can serve stateful queries for blocks and transactions created before the [Bedrock Upgrade](https://blog.oplabs.co/introducing-optimism-bedrock/).
 
-![Bedrock deployment diagram.](/img/guides/node-operators/bedrock-deployment.png)
```
Comment (sbvegan) [inline pages/builders/node-operators/overview.mdx:24]: Can we keep the diagram? I think its a nice visual

File: pages/builders/node-operators/overview.mdx (hunk 1 above)
Comment (smartcontracts) [reply pages/builders/node-operators/overview.mdx:45]: Good catch thank you

File: pages/builders/node-operators/overview.mdx (hunk 3 above)
Comment (smartcontracts) [reply pages/builders/node-operators/overview.mdx:24]: Lemme just clean it up

**Review by sbvegan — APPROVED** [review]
Looks great!

**Pull request merged**